dbos/_dbos.py

```diff
@@ -363,13 +363,13 @@ def __init__(
             check_config_consistency(name=unvalidated_config["name"])
 
         if unvalidated_config is not None:
-            self.config: ConfigFile = process_config(data=unvalidated_config)
+            self._config: ConfigFile = process_config(data=unvalidated_config)
```

Comment:
There are also uses in `new` to check out. Make sure you catch all of them!

Reply:
Are you talking about `_dbos_global_instance.config` used in 
```python
if (config is not None and _dbos_global_instance.config is not config) or (
                _dbos_global_instance.fastapi is not fastapi
            ):
                raise DBOSException(
                    f"DBOS Initialized multiple times with conflicting configuration / fastapi information"
                )
```
I didn't change that because `_config` should only be accessed from within itself.

Reply:
That is inside itself, and this is actually the one that can recurse